pkg/cmd/authLogin.go

```diff
@@ -18,35 +18,64 @@ type AuthLoginCmdValues struct {
 	bootstrap string
 }
 
-func createAuthLoginCmd(factory Factory, parentCmd *cobra.Command, rootCmdValues *RootCmdValues) (*cobra.Command, error) {
+type AuthLoginComamnd struct {
+	values       *AuthLoginCmdValues
+	cobraCommand *cobra.Command
+}
+
+// ------------------------------------------------------------------------------------------------
+// Constructors methods
+// ------------------------------------------------------------------------------------------------
+func NewAuthLoginCommand(factory Factory, authCommand GalasaCommand, rootCommand GalasaCommand) (GalasaCommand, error) {
+	cmd := new(AuthLoginComamnd)
+	err := cmd.init(factory, authCommand, rootCommand)
+	return cmd, err
+}
+
+// ------------------------------------------------------------------------------------------------
+// Public methods
+// ------------------------------------------------------------------------------------------------
+func (cmd *AuthLoginComamnd) GetName() string {
+	return COMMAND_NAME_AUTH_LOGIN
+}
+
+func (cmd *AuthLoginComamnd) GetCobraCommand() *cobra.Command {
+	return cmd.cobraCommand
+}
+
+func (cmd *AuthLoginComamnd) GetValues() interface{} {
+	return cmd.values
+}
+
+// ------------------------------------------------------------------------------------------------
+// Private methods
+// ------------------------------------------------------------------------------------------------
+func (cmd *AuthLoginComamnd) init(factory Factory, authCommand GalasaCommand, rootCommand GalasaCommand) error {
```

Comment:
This section strays from other conventions. Should there be a change to have a seperate function for building the cobra and galasa command?

Reply:
Fixed.